### ISSUE TITLE
Bug: SetUnhandledExceptionFilter in windows-sys returns fn, not Option<fn>

### ISSUE THREAD

**Issue by BlackHoleFox** [microsoft/windows-rs#1383]
### Which crate is this about?

windows-sys

### Crate version

0.28.0

### Summary

According to the [Win32 docs](https://docs.microsoft.com/en-us/windows/win32/api/errhandlingapi/nf-errhandlingapi-setunhandledexceptionfilter#return-value), the function returns `null` if there is no global exception filter already set. The actual function in `windows-sys` doesn't mirror this though, defining an unconditional function ptr as a return value:
```rust
// Inside `windows-sys`
#[cfg(all(feature = "Win32_Foundation", feature = "Win32_System_Kernel"))]
pub fn SetUnhandledExceptionFilter(lptoplevelexceptionfilter: ::core::option::Option<LPTOP_LEVEL_EXCEPTION_FILTER>) -> LPTOP_LEVEL_EXCEPTION_FILTER;
```
You can replicate this with this short snippet:
```rust
use windows_sys::Win32::System::Diagnostics::Debug::{SetUnhandledExceptionFilter, EXCEPTION_POINTERS};

fn main() {
    let old_handler = unsafe { SetUnhandledExceptionFilter(None) };
    println!("{:?}", old_handler as *const u8); // A real pointer
    assert_ne!(std::ptr::null(), old_handler as *const u8);

    let should_be_null = unsafe { SetUnhandledExceptionFilter(None) };
    println!("{:?}", should_be_null as *const u8); // 0x0
    assert_eq!(std::ptr::null(), should_be_null as *const u8);

    let fake_exception: EXCEPTION_POINTERS = unimplemented!();
    unsafe { should_be_null(&fake_exception) } // This is UB and would crash.
}
```

Interestingly the docs for the `windows` crate [have the correct definition](https://microsoft.github.io/windows-docs-rs/doc/windows/Win32/System/Diagnostics/Debug/fn.SetUnhandledExceptionFilter.html), which returns `Option<fn>`. But the definition in `windows-sys` is currently UB [per the docs on `fn`](https://doc.rust-lang.org/std/primitive.fn.html) as they are assumed to be non-null.

### Expected behavior

`SetUnhandledExceptionFilter` in `windows-sys` should be generated with `Option<fn>` as it's return type, correctly modeling the possibility of `null` being returned.

### Actual behavior

The generated return value creates undefined behavior as a `fn` pointer is `null`.

### Additional comments

Other then this, `windows-sys` has been great to work with, thanks for making it :)

**Comment by kennykerr** [comment]
This has been fixed (on GitHub) and will be available in the next version.

**Comment by kennykerr** [comment]
This has been fixed as of 0.29 of both the `windows` and `windows-sys` crates.
